app/services/importers/crime_map.rb

```diff
@@ -0,0 +1,379 @@
+require 'httparty'
+require 'json'
+
+module Importers
+  # rubocop:disable Metrics/ClassLength
+  class CrimeMap < ApplicationService
+    def initialize
+      init_zoom_okc_metro_params
+      super()
+    end
+
+    def init_zoom_okc_metro_params
+      @east = -95.86508397102726
+      @north = 36.42498633282242
+      @south = 34.142898155956104
+      @west = -99.00717381477726
+      @lat = 35.283942244389266
+      @lng = -97.43612889290226
+      @zoom = 9
+    end
+
+    def perform
+      date_range = (Date.parse(ENV.fetch('LEXUS_NEXUS_START', '2020-01-01'))..Time.zone.now)
+      date_range.step(3).each do |date|
+        @start_date = date
+        @end_date = date + 1.week
+        save_crimes
+      end
```

Comment:
it turns out we can kinda get in historic using the crime map
The results I get are very inconsistent. For September we got in over a thousand distinct using the pdf but only got in a fraction of them using the crime map, but for some moths we get many many more.

Reply:
I checked Moore, Norman, Midwest City, and Edmond after doing an import back to 2020 and they all look somewhat consistent compared to OKCPD, who jumps between 1000 to 100 to 30. So hopefully between the two imports we will have a decent dataset, but we could do more analysis.

Reply:
There are also a lot of null `incident_at`s I could look into more
```pgsql
select agency, date_trunc('month', incident_at), count(*) from lexus_nexus2_crimes
group by agency, date_trunc('month', incident_at)
order by agency, date_trunc('month', incident_at) desc
```